hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -157,15 +159,48 @@ def setUp(self):
 
         s.set_elements(elements)
         s.add_lines(xray_lines)
+        s.axes_manager[0].scale = 0.5
+        s.axes_manager[1].scale = 0.5
         self.signal = s
 
     def test_quant_lorimer(self):
         s = self.signal
+        method = 'CL'
         kfactors = [1, 2.0009344042484134]
+        composition_units = 'weight'
         intensities = s.get_lines_intensity()
-        res = s.quantification(intensities, kfactors)
-        nt.assert_true(np.allclose(res[0].data, np.array(
-                       [22.70779, 22.70779]), atol=1e-3))
+        res = s.quantification(intensities, method, kfactors, composition_units)
+        np.testing.assert_allclose(res[0].data, np.array([[22.70779, 22.70779],
+                    [22.70779, 22.70779]]), atol=1e-3)
```

Comment:
Sorry, I think you need to specify rtol=1e-3 also, in all of these allclose cases... This defaults to 1e-7 but since you don't have that many decimal places that's failing Travis